### DISCUSSION TITLE
README.md instructions - running the web app

### DISCUSSION THREAD

**Discussion by jimmyshephard** [tutao/tutanota#8179] [General]
I want to contribute, but I have had issues running the application. The `README.md` instructions say I should be able to node server, but there's no server file in the build directory. What's the correct way to run the web app? Also, the `README` says `WASM2JS` is included in `Emscripten` but I did not find this to be the case, perhaps these instructions need an update?

**Comment by charlag** [comment]
Hi, we have updated the instructions, you can use any http server to serve the web client.
As for wasm2js it is included at least in the Ubuntu package but it might vary on distribution
